src/main/webapp/login.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Login Page</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="login_script.js"></script>
+  </head>
+  <body>
+    <div id="login_form">
+        <h3>Log in below:</h3>
+        <form id="signup">
+            <label for="email">Email:</label><br>
+            <input type="text" id="email" name="email"><br>
+            <label for="pwd">Password:</label><br>
+            <input type="text" id="pwd" name="pwd"><br><br>
+            <input type="submit" value="Login" onclick="putCookie(document.getElementsByTagName('form'));">
```

Comment:
This action should be done in the Server. Refer to Week 2 walkthrough.

1. Remove this onclick action.
2. Create a Servlet for login, e.g. `LoginServlet.java`. The Servlet should implement `doPost()`. Here, you should extract and validate the email, i.e., decide if it corresponds to a TA. See Step 4 of walkthrough.
3. The Server sends cookies using `response.addCookie(...)`. See https://beginnersbook.com/2013/05/servlet-cookies/
4. Redirect is done with `response.sendRedirect("/index.html");`. See Step 8 of walkthrough.
5. Connect this form with the Servlet using `method` and `action`. See Step 5 of walkthrough.

Reply:
updated in newest commit